sponsorship.md

```diff
@@ -26,6 +26,28 @@ Use of donations is decided transparently via the Technical Steering Committee
 
 </div>
 
+<div id="sponsorship-tiers">
+
+## Sponsorship Tiers
+
+- **Platinum: 10,000 USD/month**
+- **Gold: 5,000 USD/month**
+- **Silver: 1,000 USD/month**
+- **Bronze: 100 USD/month**
+
+By sponsoring through one of the tiers **your logo/name will be displayed at [servo.org](https://servo.org/)**, provided that it doesn't violate [Servo's Code of Conduct](https://servo.org/coc).
```

Comment:
Maybe we can rewrite this to make it sound less transactional:

```suggestion
Sponsors in the Bronze tier and above will be acknowledged on [servo.org](https://servo.org/) with **their logo or name**.
```

Not sure if we want to mention the CoC as the TSC will review anyway and look at more than just the CoC.